translations/component-templates/template-this/pt-br.yaml

```
@@ -0,0 +1,6 @@
+title: Usando properties próprias
+description: >
+  Se uma property é definida no arquivo JavaScript desse componente, use 
+  If a property is defined in the JavaScript file for this component's template, use '<code>'this'</code>' quando estiver fazendo uma referência.
```

I hope I combined the two lines correctly. (Wasn't sure if `use` is the correct imperative form in `pt-br`.)

```suggestion
  Se uma property é definida no arquivo JavaScript desse componente, use '<code>'this'</code>' quando estiver fazendo uma referência.
```

---

You got it right, it is correct.